crates/solidity/inputs/language/src/definition.rs

```diff
@@ -0,0 +1,3854 @@
+                                Variant(
+                                    name = Override,
+                                    fields = (specifier = Required(NonTerminal(OverrideSpecifier)))
+                                ),
+                                Variant(
+                                    name = Constant,
+                                    disabled_in = "0.5.0",
+                                    fields = (keyword = Required(Terminal([ConstantKeyword])))
+                                ),
+                                Variant(
+                                    name = External,
+                                    fields = (keyword = Required(Terminal([ExternalKeyword])))
+                                ),
+                                Variant(
+                                    name = Internal,
+                                    fields = (keyword = Required(Terminal([InternalKeyword])))
+                                ),
+                                Variant(
+                                    name = Payable,
+                                    fields = (keyword = Required(Terminal([PayableKeyword])))
+                                ),
+                                Variant(
+                                    name = Private,
+                                    fields = (keyword = Required(Terminal([PrivateKeyword])))
+                                ),
+                                Variant(
+                                    name = Public,
+                                    fields = (keyword = Required(Terminal([PublicKeyword])))
+                                ),
+                                Variant(
+                                    name = Pure,
+                                    fields = (keyword = Required(Terminal([PureKeyword])))
+                                ),
+                                Variant(
+                                    name = View,
+                                    fields = (keyword = Required(Terminal([ViewKeyword])))
+                                ),
+                                Variant(
+                                    name = Virtual,
+                                    enabled_in = "0.6.0",
+                                    fields = (keyword = Required(Terminal([VirtualKeyword])))
+                                )
+                            ]
+                        ),
+                        Struct(
+                            name = OverrideSpecifier,
+                            fields = (
+                                override_keyword = Required(Terminal([OverrideKeyword])),
+                                open_paren = Required(Terminal([OpenParen])),
+                                overridden = Required(NonTerminal(OverridePaths)),
+                                close_paren = Required(Terminal([CloseParen]))
+                            )
+                        ),
+                        Separated(
```

Comment:
`SeparatedBy`?

Reply:
I suggest using `Separated` since it describes the item being separated (higher importance to describe/annotate). The term `SeparatedBy` refers to the separator itself (lower importance). This is also reflected in the backing rust type `SeparatedItem`, instead of `SeparatedByItem`.

For example in `Parameters`, the focus here is on the `Parameter` being separated, not the `Comma`. This is the thing we would typically extract/use in further passes.